docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -41,35 +41,13 @@ Result:
 └──────────────────────────────────────────────────┘
 ```
 
-Query:
-
-```sql
-CREATE TABLE table_map (a Map(String, UInt64)) ENGINE = MergeTree() ORDER BY a;
-INSERT INTO table_map SELECT map('key1', number, 'key2', number * 2) FROM numbers(3);
-SELECT a['key2'] FROM table_map;
-```
-
-Result:
-
-```text
-┌─arrayElement(a, 'key2')─┐
-│                       0 │
-│                       2 │
-│                       4 │
-└─────────────────────────┘
-```
-
-**See Also**
-
-- [Map(key, value)](../data-types/map.md) data type
-
 ## mapFromArrays
 
-Merges an [Array](../data-types/array.md) of keys and an [Array](../data-types/array.md) of values into a [Map(key, value)](../data-types/map.md). Notice that the second argument could also be a [Map](../data-types/map.md), thus it is casted to an Array when executing.
-
-
-The function is a more convenient alternative to `CAST((key_array, value_array_or_map), 'Map(key_type, value_type)')`. For example, instead of writing `CAST((['aa', 'bb'], [4, 5]), 'Map(String, UInt32)')`, you can write `mapFromArrays(['aa', 'bb'], [4, 5])`.
+Creates a map from an array of keys and an array of values.
+The second argument can also be a map, it will be casted during execution to an array.
 
+The function is a convenient alternative to syntax `CAST((key_array, value_array_or_map), 'Map(key_type, value_type)')`.
+For example, instead of writing `CAST((['aa', 'bb'], [4, 5]), 'Map(String, UInt32)')`, you can write `mapFromArrays(['aa', 'bb'], [4, 5])`.
```

Comment:
Above "Maps are internally implemented as:" `Array(Tuple(String,UInt32))`
Why not `CAST([('aa',4)('bb',5)] AS 'Map(String, UInt32)')` then?

Reply:
That's an alternative syntax, I'll add it.